web/src/input_pill.ts

```diff
@@ -430,6 +470,15 @@ export function create<T>(opts: InputPillCreateOptions<T>): InputPillContainer<T
         // when the "×" is clicked on a pill, it should delete that pill and then
         // select the next pill (or input).
         store.$parent.on("click", ".exit", function (e) {
+            // TODO(evy) this really should live in search land, not here, but
+            // I don't know a good way to do that.
```

Comment:
I even had to put the click handler here because `funcs` isn't public to other files.

Reply:
I think this is also probably fine to live here for now, just add a comment as with my suggestion on the other part of the commit, that both notes a TODO and also explains what `$user_pill_container.length` means clearly to someone who doesn't know anything about search.

Maybe best would be inside the `$user_pill_container.length` block, a comment like>

> // The user-pill-container container class is used exclusively for group-DM search pills, where multiple user pills sit inside a larger pill. The exit icons in those individual user pills should remove just that pill, not the outer pill.
> // TODO: Figure out how to move this code into `search_pill.ts`.

Reply:
Cool, thanks, did that!